source/base/parameter_handler.cc

```diff
@@ -736,6 +736,42 @@ ParameterHandler::parse_input_from_json(std::istream &in,
 
 
 
+void
+ParameterHandler::create_default_input_file(const std::string filename,
+                                            const bool        sort_alphabetical,
+                                            const bool short_version) const
+{
+  std::ofstream out(filename);
+  Assert(out, ExcIO());
+
+  std::string file_ending = filename.substr(filename.find_last_of(".") + 1);
+
+  if (file_ending == "prm")
+    if (short_version)
+      print_parameters(out, ParameterHandler::ShortText, sort_alphabetical);
+    else
+      print_parameters(out, ParameterHandler::Text, sort_alphabetical);
```

Comment:
If we would make the change described above, we could write:
```suggestion
    print_parameters(out, ParameterHandler::Text | short_version, sort_alphabetical);
```

(Of coarse, in this case `short_version` is not a bool anymore.)

Reply:
I think for this PR it is important to agree on the interface of the function, the implementation is something that can be changed later given that `OutputStyle` is work in progress. For the interface, I believe `bool` is the correct type. If another type is needed to communicate with `OutputStyle`, I think the type should be transferred within this function.

Reply:
I would also not use a Boolean for `sort_alfabetical`. If we go towards the route of the enum, also the alphabetical sorting is part of the output style, and could be specified by a bit in the enum.